eeco/tests/test_costs.py

```diff
@@ -210,14 +279,14 @@ def test_create_charge_array(
             input_dir + "billing_energy_peak.csv",
             "15m",
             {
-                "electric_energy_off-peak_20240710_20240710_0": np.concatenate(
+                "electric_energy_offpeak_20240710_20240710_0": np.concatenate(
```

Comment:
Following off of my comments in `costs.py`, I would rather not change this functionality as could break previous code.